doc/api/axes_api.rst

```diff
@@ -1,18 +1,30 @@
-================
- ``Axes`` class
-================
+====
+axes
+====
+
 .. currentmodule:: matplotlib.axes
 
 .. autoclass:: Axes
    :no-members:
    :no-undoc-members:
+   :show-inheritance:
 
 .. contents:: Table of Contents
    :depth: 2
    :local:
    :backlinks: entry
    :class: multicol-toc
 
+Subplots
```

Comment:
This appears in the TOC as a level able "Plotting", so I think needs a lower decoration level:

https://10464-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/axes_api.html

Reply:
I tried to change the header hierarchy without checking the result... and I see now that  didn't work and that I didn't know how headers works in rst. I tried to lower the subplot level and that level stayed the same but everything else toggled its level... Tries again.

Reply:
Not sure this is going the right way. This whole page was documenting the `Axes` class. You've now changed it to the `axes` module and inserted the subplots stuff in between the class description and the list of functions (the sections here are used to group the `Axes` methods by topic).

Reply:
But why should there be a special page for documenting the Axes class instead of the axes module?

But the page can of course be restructured in same way.

Reply:
The `Axes` class is so large and fundamental, that it might have a dedicated page. But I'm not saying it has to be that way. I also laid out a possible structure if the file should contain the whole module.